pkg/types/installconfig.go

```diff
@@ -43,6 +44,7 @@ var (
 	// hidden-but-supported platform names. This list isn't presented
 	// to the user in the interactive wizard.
 	HiddenPlatformNames = []string{
+		alibabacloud.Name,
```

Comment:
Let's move this out of hiddenplatformnames  to the supported platforms since we are targeting 4.10 release.